gcloud/bigquery/job.py

```diff
@@ -919,3 +931,278 @@ def _scrub_local_properties(self, cleaned):
                 assert dest_remote['projectId'] == self.project
                 dataset = self._client.dataset(dest_remote['datasetId'])
                 self.destination_table = dataset.table(dest_remote['tableId'])
+
+
+class _SyncQueryConfiguration(object):
+    """User-settable configuration options for synchronous query jobs."""
+    # None -> use server default.
```

Comment:
That is correct.  The comment vs. docstring follows the same pattern as the other private/implementation detail `*Config` classes.